TripalCultivate/TripalCultivate-Theme
# Updates twig template

This PR is an update to twig template to include block area that will show context menu options to modify a tile. Menu option is activated by hovering a tile.

![image](https://github.com/TripalCultivate/TripalCultivate-Theme/assets/15472253/52bc4c2b-b1c4-4581-b8c8-f2ae9f4fb586)

Additional improvements in this PR include:
1. Favicon
2. Default tile background color.

To Test.
1. Clone this Repo and switch to branch **small-improvments**
2. Create a clone:

```
> docker build --tag=trpcultivate:11 --build-arg drupalversion=10.2.x-dev --build-arg phpversion=8.3 ./

> docker run --publish=80:80 -tid --name=theme11 trpcultivate:11 && docker exec theme11 service postgresql restart
```

3. Load frontpage and login as admin

**TEST DEFAULT TILE COLOUR:**
1. Access my site/admin/appearance and click Settings link 
![image](https://github.com/TripalCultivate/TripalCultivate-Theme/assets/15472253/e1a8e034-0de4-4128-9561-151e282c1717)

2. Scroll down to Olivero Utilities and select a Olivero Color Scheme.
![image](https://github.com/TripalCultivate/TripalCultivate-Theme/assets/15472253/30cb30c3-ef16-41e3-bfc0-855eac9d8df4)
3. Click Save configuration and inspect home page and see if the header section and tiles have the same/similar color.

**TEST FAVICON:**
1. In the homepage, ispect the tab area of the Browser for the Tripal Cultivate Icon
![image](https://github.com/TripalCultivate/TripalCultivate-Theme/assets/15472253/ddbf779c-9357-4798-8d4f-8ae0982cdb1b)

**TEST CONTEXT MENU:**
1. Access my site/admin/content and click Blocks tab
![image](https://github.com/TripalCultivate/TripalCultivate-Theme/assets/15472253/b5e95225-64ad-441e-bef1-11334cbe0358)
2. Click +Add content block button. In the Add content block click Tripal Cultivate Theme Tile Block to load content entry form.
3. Create a content by adding Description (Use the value entered to identify content), Tile title, Url/link and Link text, Image and a Hex colour code.
4. Click Save and access my site/admin/structure/block.
5. Scroll down to Tripal Cultivate Tile Regions (at the bottom of the page).
![image](https://github.com/TripalCultivate/TripalCultivate-Theme/assets/15472253/187ff48f-3766-4f97-a88a-729ad3d17f29)
6. Click on Place block button next to the region. A window pop up showing all content created will appear. Select the content created in step 3 using the value entered in description field. Click Place block.
7. Repeat until all regions have the content added.
8. Click Save block and refresh homepage.
9. Hover mouse to tiles to reveal context menu.
![image](https://github.com/TripalCultivate/TripalCultivate-Theme/assets/15472253/52bc4c2b-b1c4-4581-b8c8-f2ae9f4fb586)
